docs/src/samples/monitoring/grafana-configmap.yaml

```diff
@@ -919,7 +919,7 @@ data:
             {
               "datasource": "${DS_PROMETHEUS}",
               "editorMode": "code",
-              "expr": "max(cnpg_pg_stat_replication_flush_lag_seconds{namespace=~\"$namespace\",pod=~\"$cluster-.*\"})",
+              "expr": "max(cnpg_pg_stat_replication_flush_lag_seconds{namespace=~\"$namespace\",pod=~\"$cluster-[0-99]+$\"})",
```

Comment:
This Regex is incorrect. Could you replace it with: `-[1-9][0-9]?$`.

Could you replace it in both documents?

Reply:
I would also be fine with `-[0-9]+$` for the sake of simplicity.